app/views/application/_radius_details.html.erb

```diff
@@ -18,14 +18,14 @@
     <h3> London: </h3>
     <ul class="plain-list" id="london-radius-ips">
       <% london_ips.each do |ip| %>
-        <li><p class="govuk-body"> <%= ip %></li>
+        <li><p class="govuk-body"><span class="ip-address"><% ip.split('.')[0...-1].each do |chunk| -%><%= chunk -%><span class="ip-address-separator">.</span><% end -%><%= ip.split('.').last %></span></li>
```

Comment:
This IP formatting seems like it would be better defined in a ERB partial as it's repeated on line 28.

Reply:
As discussed this is being resolved. :+1: